src/helpers/sdkVkId.ts

```diff
@@ -11,3 +11,5 @@ export const vkIdConfig = VKID.Config.init({
 	codeChallenge: String(generateCodeChallenge()),
 	scope: "email phone",
 });
```

Comment:
VKID.Config.init({
...
responseMode: VKID.ConfigResponseMode.Callback,
...
})

can you replace VK SDK config to SignInForm and SignUpForm components with  responseMode field?